Default partial parameters fail the linter

Recently, we added support for parameters in partials, including default values (https://github.com/gravitational/docs/pull/171). However, if you create a partial with a default parameter assignment, e.g.:

```
{{ myvar="myval" }}

This partial uses {{ myvar }}.
```

..the page will render as expected when you run `yarn dev`, but running `npx remark` will lead to the following error:

```
         1:1  error    Could not parse expression with acorn: Shorthand property assignments are valid only in destructuring patterns
```

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "lint": "yarn base:eslint --fix && yarn base:prettier --write -l",
     "lint-check": "yarn base:eslint && yarn base:prettier --check",
     "typecheck": "tsc --noEmit --skipLibCheck --incremental --tsBuildInfoFile node_modules/.cache/tsc/tsbuildinfo --project .",
-    "markdown-lint": "remark 'content/**/docs/pages/**/*.mdx' --frail",
+    "markdown-lint": "remark 'content/**/docs/pages/**/*.mdx' --frail --ignore-pattern 'content/*/docs/pages/includes/*' --silently-ignore",
     "markdown-lint-external-links": "WITH_EXTERNAL_LINKS=true yarn markdown-lint",
     "markdown-fix": "FIX=true yarn markdown-lint -o"
   },

